server/mhn/templates/base.html

```diff
@@ -48,10 +48,14 @@
 			 <li class="has-dropdown">
                         <a href="/ui/chart">Charts</a>
                         <ul class="dropdown">
-                            <li><a href="/image/top_passwords.svg">Kippo Top Passwords</a></li>
```

Comment:
I have made this change because I want kippo and cowrie separated. 2 honeypots, 2 separated statistics. Anyway you can change this how you want, no problem. The important is that cowrie is in MHN! :P